src/__tests__/createTask.js

```diff
@@ -0,0 +1,50 @@
+import { jsxEmptyExpression } from '@babel/types'; // eslint-disable-line
+import createTask from '../functionalities/createTask.js';
+import deleteTask from '../functionalities/deleteTask.js';
+import tasks from '../__mocks__/createTask.js';
+
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
+
+const dom = new JSDOM(`<!DOCTYPE html><div id='main'></div>`); // eslint-disable-line
+const document = dom.window.document; // eslint-disable-line
+
+describe('Test add method', () => {
```

Comment:
- [ ] You have only tested the `addTask` and `removeTask` funtionality, please remember that you have to test the majority of the [pure funtions](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwiAtriui__zAhW7h_0HHQcLD5wQFnoECA8QAw&url=https%3A%2F%2Fwww.sitepoint.com%2Ffunctional-programming-pure-functions%2F&usg=AOvVaw06n7VkqKrBdswbXrmU9T9H) 🚀

Reply:
Hi Andres, thank you for your review.

Just one comment. This review is for the first project submission (https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md)
And Here it says this:
 "Create a test file ([..].test.js) for a file containing the **add item** and **delete item** functions that you want to test."

I think for the next part of this week's project we need to test the other functions as well.